js/credits.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function(){
+    var $id=$('#select-lang');
+    var $a=$('#dropdownMenu');
+    $a.css('color','white');
```

Comment:
Please try to add css to the css files. Otherwise we might not be able to find the place to edit the style in the future and ask ourselves where this color comes from.

Reply:
@niccokunzmann I have included css for this particular element here because including it in css file is not working.
When I include the CSS for this in js then only it is getting applied. I guess this is because js overwrites embedded CSS .